uploads/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
+!.gitignore
```

Comment:
Sorry to belabor this further, but we shouldn't need to unignore a `.gitignore` file??

I tried it with:

```
*
!*/
!agencies/cst-*.png
```

And everything still works as expected, including changes to the `.gitignore` file:

![image](https://github.com/user-attachments/assets/22bcc33a-1fdb-4eda-ab05-2d98ebf3b4f2)

![image](https://github.com/user-attachments/assets/3725cadf-89ec-4fc9-8c4a-d5bff862071f)

Reply:
I might be wrong about this, because the `.gitignore` was already committed?

Reply:
Not at all, thanks for the suggestion. I just tried it and I also see that it works as expected, I'll make this update 👍